matlab-src/TorqueBalancing_js/centroidalTransformationT_TDot.m

```diff
@@ -27,12 +27,11 @@
 dMb     = [zeros(3),Sf(mdr)';
            Sf(mdr),zeros(3)];
 
-invMb   =  eye(6)/Mb;
-inv_dMb = -invMb*dMb*invMb;
+inv_dMb = -Mb\dMb/Mb;
```

Comment:
@Ganimed if you want to discuss a specific line of the pull request, please do it commenting a specific line, to avoid polluting the main thread, thanks : https://help.github.com/articles/commenting-on-the-diff-of-a-pull-request/ .